README.md

```diff
@@ -19,18 +20,106 @@ Coupler Specific Shared Development
 |---------------------:|:-----------------------------------------------|
 | **Documentation**    | [![dev][docs-dev-img]][docs-dev-url] [![docs build][docs-bld-img]][docs-bld-url]|
 | **Unit Tests**       | [![unit tests][unit-tests-img]][unit-tests-url] [![codecov][codecov-img]][codecov-url]|
-
-Provides coupled system time stepping control and support for mapping import and export
-boundary information between components.
+|||
 
 Recommended Julia Version: Stable release v1.10.1. CI no longer tests earlier versions of Julia.
 
-## Start Up
-Before starting Julia, ensure your environment is properly set up. This includes loading the correct Julia version (specified in the `Project.toml` file), modules and setting the correct environment variables.
+# Running AMIP
+Here we will focus on the AMIP experiment, which uses the environment in the `experiments/ClimaEarth/` subdirectory of ClimaCoupler.jl
+The first step to do this is to install all required packages for the environment using the following Julia command:
+```julia
+julia --project=experiments/ClimaEarth -E "using Pkg; Pkg.instantiate(); Pkg.build()"
+```
+
+Now you're ready to run the experiment, which uses the `run_amip.jl` driver. To run interactively:
+```
+julia --project=experiments/ClimaEarth
+julia> include("run_amip.jl")
+```
+
+Or to run from the terminal:
+```julia
+julia --project=experiments/ClimaEarth run_amip.jl
+```
+
+When running from the terminal, you can also specify a configuration file to use for the simulation setup, and a job ID to keep track of this run's output.
+Existing configuration files are specified in the `config/` directory within ClimaCoupler.jl.
+For example, to run a coarse AMIP run using Float64, you could use the following command:
+```julia
+julia --project=experiments/ClimaEarth run_amip.jl --config_file config/ci_configs/coarse_single_ft64.jl --job_id coarse_single_ft64
```

Comment:
I think that when we provide examples we should not use CI stuff. CI has a different goal than pedagogy.

Reply:
I think it's helpful to provide an example showing how to use a non-default configuration because this is something people in our group will want to do and have asked about in the past

Reply:
I agree: we should have a proper example, not something we also use for CI (but we can worry about this in the future)

Reply:
I recently opened an issue about changing our default config file - we can revisit this when we address that https://github.com/CliMA/ClimaCoupler.jl/issues/1002